README.md

```diff
@@ -1,6 +1,6 @@
 # [Transformers4Rec](https://github.com/NVIDIA-Merlin/Transformers4Rec/) | [Documentation](https://nvidia-merlin.github.io/Transformers4Rec/main/)
 <p align="left">
-    <a href="https://github.com/NVIDIA-Merlin/Transformers4Rec/actions/workflows/ci.yml">
+    <a href="https://github.com/NVIDIA-Merlin/Transformers4Rec/actions/workflows/tests.yml">
         <img alt="Build" src="https://img.shields.io/github/checks-status/NVIDIA-Merlin/Transformers4Rec/main">
```

Comment:
The image at this url just says 'Checks: Pending'  - can we use the github actions one instead? https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Like https://github.com/NVIDIA-Merlin/Transformers4Rec/actions/workflows/ci.yml/badge.svg?branch=main currently - or https://github.com/NVIDIA-Merlin/Transformers4Rec/actions/workflows/tests.yml/badge.svg?branch=main after this change ?